File: AtlasWeb/Core/Modules/Tibbi_Surec/Ameliyat_Modulu/SurgeryRejectReasonFormViewModel.ts
//$B8BC1FFB
import { BaseViewModel } from "NebulaClient/Model/BaseViewModel";
import { SurgeryRejectReason } from "NebulaClient/Model/AtlasClientModel";

export class SurgeryRejectReasonFormViewModel extends BaseViewModel {
    public _SurgeryRejectReason: SurgeryRejectReason = new SurgeryRejectReason();
}

File: AtlasWeb/Core/Modules/Tibbi_Surec/Ameliyat_Modulu/SurgeryRejectReasonFormViewModel.cs
//$AA05E2FD
using System;
using System.Linq;
using Core.Models;
using Infrastructure.Filters;
using Infrastructure.Models;
using TTInstanceManagement;
using TTObjectClasses;
using TTDefinitionManagement;
using TTUtils;

namespace Core.Controllers
{
    public partial class SurgeryRejectReasonServiceController
    {
        partial void PreScript_SurgeryRejectReasonForm(SurgeryRejectReasonFormViewModel viewModel, SurgeryRejectReason surgeryRejectReason, TTObjectContext objectContext)
        {
            
        }
    }
}

namespace Core.Models
{
    public partial class SurgeryRejectReasonFormViewModel: BaseViewModel
    {
    }
}
